conf.d/elastic.yaml.example

```diff
@@ -15,10 +15,35 @@ instances:
   # This parameter was also called `is_external` and you can still use it but it
   # will be removed in version 6.
   #
+  # If you enable the "pshard_stats" flag, statistics over primary shards
+  # will be collected by the check and sent to the backend with the
+  # 'elasticsearch.primary' prefix. It is particularly useful if you want to
+  # get certain metrics without taking replicas into account. For instance,
+  # 'elasticsearch.primaries.docs.count` will give you the total number of
+  # documents in your indexes WITHOUT counting duplicates due to the existence
+  # of replica shards in your ES cluster
+  #
+  # The "shard_level_metrics" flag enables metrics and service checks on a per-
+  # shard basis (all the information is fetched under the /_stats?level=shards
+  # endpoint). The metrics and service check sent for each shard are named as
+  # such: elasticsearch.shard.metric.name .
```

Comment:
you could add a line about how the metrics are tagged, so that people know how to graph per shard.